lib/rules/sort-imports.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
sort-imports is "suggestion" but sort-keys is "style"-- feels inconsistent.

Reply:
Seems like they both should be suggestion.